src/App.js

```diff
@@ -2,14 +2,20 @@
 import React from 'react';
 import Calculator from './components/Calculator';
 
-class App extends React.Component {
-  render() {
-    return (
-      <>
-        <Calculator />
-      </>
-    );
-  }
-}
+// class App extends React.Component {
+//   render() {
+//     return (
+//       <>
+//         <Calculator />
+//       </>
+//     );
+//   }
+// }
```

Comment:
- [ ] As pointed in the JavaScript requirements of the project in JavaScript best practice's kindly remove the commented code, you can always check old commits if you want to see how it was before

![image](https://user-images.githubusercontent.com/73140461/154558009-566faae0-0602-4890-96c0-1900669bb17d.png)